plugins/polio/js/src/forms/ScopeForm.tsx

```diff
@@ -17,6 +17,36 @@ type Round = {
 type Values = {
     separate_scopes_per_round?: boolean;
     rounds: Round[];
+    initial_org_unit: number;
+};
+
+type Scope = {
```

Comment:
This doesn't look like it's part of this PR

Reply:
solved